src/domain/app/layout/PageWrapper.tsx

```diff
@@ -31,6 +38,16 @@ const PageWrapper: React.FC<PageWrapperProps> = ({
     title: translatedTitle,
   };
 
+  // Track page view
+  useDeepCompareEffect(() => {
+    if (consent.tracking) {
+      trackPageView({
+        documentTitle: translatedTitle,
+        href: window.location.href,
+      });
+    }
+  }, [{ pathname: location.pathname, search: location.search }]);
```

Comment:
is useDeepCompareEffect needed here?

Reply:
Does this have exactly the same behaviour?
```
React.useEffect(() => {}, [location.pathname, location.search])
```

Reply:
You are absolutely right.  Changed it to 
```
React.useEffect(() => {
    if (consent.tracking) {
      trackPageView({
        documentTitle: translatedTitle,
        href: window.location.href,
      });
    }
    // eslint-disable-next-line react-hooks/exhaustive-deps
  }, [consent.tracking, location.pathname, location.search]);
  ```
  so the page is tracked also after accepting cookies